### PR TITLE
chore: implement HumanString for About and Applications sections of env show

### PR DESCRIPTION
**Pull request by huanjani**
Implemented human output for basic parts of env show. Followed styling of `app show` and `project show`.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by huanjani** [comment]
example manual output:
```
$ ecs-preview project show

? Which project's applications would you like to show? dinder
About

  Name              dinder
  URI               

Environments

  Name              AccountID           Region
  test              327148513104        us-west-2

Applications

  Name              Type
  api               Load Balanced Web App
  webapp            Load Balanced Web App
$

File: internal/pkg/describe/env.go [hunk 1]
```diff
@@ -64,17 +67,32 @@ func (e *EnvDescriber) Describe() (*EnvDescription, error) {
 	}, nil
 }
 
-// JSONString returns the stringified WebApp struct with json format.
-func (w *EnvDescription) JSONString() (string, error) {
-	b, err := json.Marshal(w)
+// JSONString returns the stringified EnvDescription struct with json format.
```
Comment (SoManyHs) [inline internal/pkg/describe/env.go:70]: lol good catch :)

File: internal/pkg/describe/env.go [hunk 2]
```diff
@@ -64,17 +67,32 @@ func (e *EnvDescriber) Describe() (*EnvDescription, error) {
...
 	return fmt.Sprintf("%s\n", b), nil
 }
 
-// HumanString returns the stringified WebApp struct with human readable format.
-func (w *EnvDescription) HumanString() string {
-	// TODO
-	return ""
+// HumanString returns the stringified EnvDescription struct with human readable format.
+func (e *EnvDescription) HumanString() string {
+	// HumanString returns the stringified WebApp struct with human readable format.
```
Comment (SoManyHs) [inline internal/pkg/describe/env.go:81]: nit: delete this line?

Comment (huanjani) [reply internal/pkg/describe/env.go:81]: Ugh, oops, yes!

**Comment by efekarakus** [comment]
a random nit but would you mind changing the PR type from "build:" to "chore:" (https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type)

**Comment by huanjani** [comment]
> a random nit but would you mind changing the PR type from "build:" to "chore:" (https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type)

Done. Thanks for always including references. That page goes beyond the conventional commits page I had been using. Super helpful!

**Pull request merged**